supervisor/config.py

```diff
@@ -57,13 +59,19 @@ def __init__(self):
         super().__init__(FILE_HASSIO_CONFIG, SCHEMA_SUPERVISOR_CONFIG)
 
     @property
-    def timezone(self) -> str:
+    def timezone(self) -> Optional[str]:
         """Return system timezone."""
-        return self._data[ATTR_TIMEZONE]
+        timezone = self._data.get(ATTR_TIMEZONE)
+        if timezone != _UTC:
+            return timezone
+        self._data.pop(ATTR_TIMEZONE, None)
```

Comment:
Can you add a comment why we would pop `UTC` from `self._data` here. Is this part of a migration? If so, should we also save it ?

Reply:
I think we can stay with this code, the reason why I changed it yesterday. Make no sense to store UTC because we are anyway fallback to system default or if nothing is set, to UTC.